tools/create-trials-index.js

```diff
@@ -322,7 +319,7 @@ function bulkIndexEntities(entities, index, indexType) {
   return result;
 }
 
-function indexModel(model, index, indexType, fetchOptions) {
+function indexModel(model, index, indexType, _queryParams, fetchOptions) {
   return model.count().then((modelCount) => {
     const bufferLength = 1000;
```

Comment:
I think `bufferLength` is clearer than `limit` because it's more specific. A variable `limit` can be many things (e.g. maybe we're just indexing `limit` entities?), but `bufferLength` tells you it's actually a buffer. Reading it again now, should probably have named it to `batchSize` or similar. Will do this :+1: